src/operators.jl

```diff
@@ -35,8 +35,6 @@ jointerms(terms1::AbstractArray{<:Term}, terms2::AbstractArray{<:Term}) = merges
 # definitions without resorting to promotion:
 MP.multconstant(v::Monomial, α) = Term(α, v)
 MP.multconstant(v::Variable, α) = Term(α, Monomial(v))
```

Comment:
I though it was the above two lines that could be deleted but I could be wrong

Reply:
You're right, thanks.